app.py

```diff
@@ -18,13 +18,18 @@ def __init__(self):
 
         self.set_titlebar(header)
 
+        # TODO: layout would paned or stack work better than flowbox?
         flowbox = Gtk.FlowBox()
         flowbox.set_valign(Gtk.Align.START)
         flowbox.set_max_children_per_line(30)
         flowbox.set_selection_mode(Gtk.SelectionMode.NONE)
 
+        # TODO: add input for the shift
+        # Gtk.Scale or Gtk.SpinButton?
+
         # start decoder
         self.message_to_decode = Gtk.TextView()
+        self.message_to_decode.set_size_request(300,50)
```

Comment:
This sets the size of the text box so it's large enough to make it easier to work with.